bootiful-music.jh

```diff
@@ -0,0 +1,40 @@
+application {
+  config {
+    baseName bootifulmusic,
+    applicationType monolith,
+    packageName com.jhipster.demo.bootifulmusic,
+    databaseType neo4j,
+    devDatabaseType neo4j,
+    prodDatabaseType neo4j,
+    enableHibernateCache false,
+    testFrameworks [protractor]
+  }
+  entities *
+}
+
+entity Artist {
+	name String required
+}
+
+entity Genre {
+	name String required
+}
+
+entity Track {
+	name String required
+}
+
+entity Album {
+	name String required
+}
+
+relationship OneToOne {
+	Album{artist(name)} to Artist
```

Comment:
Actually, an artist might have released multiple albums, and an album might be a collaboration from multiple artists, so ManyToMany would be more accurate ;)

Reply:
True, just wanted to keep it simple and yes, I assume there is one artist for an album, no collaboration allowed right now :)